### DISCUSSION TITLE
[Question]: How Do I Remove The Footer Or Edit It?

### DISCUSSION THREAD

**Discussion by Prrimcord** [danny-avila/LibreChat#1052] [Q&A] [answered]
### Contact Details

rajivgautam2008@gmail.com

### What is your question?

I want to edit the footer or delete it completely so if anybody can help i would appreciate it.

**Comment by fuegovic** [comment]
The footer itself is located here: `.\client\src\components\Input\Footer.tsx`

The text form the footer is translated in various languages. The translation files are here: `.\client\src\localization\languages` and the text you're looking for is at the line starting with `com_ui_pay_per_call:`

**Comment by danny-avila** [comment, marked as answer]
As of this PR: https://github.com/danny-avila/LibreChat/pull/1098

In your .env file add the following env variable
```bash
CUSTOM_FOOTER="My custom footer"
```

You can even make it empty to "remove" the footer
```bash
CUSTOM_FOOTER=""
```

**Comment by m3xw3ll** [comment]
Somehow it is possible to use links in the footer like in the example. If I want to create a custom footer with a link in the .env file:
`CUSTOM_FOOTER="<a href='www.google.de' target='_blank'>Go to Google</a>"`

Will just add this text to the footer, not formatted as HTML link. Can you help me?

**Reply by jcbartle** [reply]
You can't use HTML in the CUSTOM_FOOTER environment variable, but you can use Markdown.
